src/scripts/common.js

```diff
@@ -514,10 +510,16 @@ window.togglbutton = {
 
   // Make button corresponding to 'entry' active, if any. ; otherwise inactive.
   updateTimerLink: function (entry) {
-    const current = togglbutton.links.find(
-      button => invokeIfFunction(button.params.description) === entry.description);
+    if (!entry) {
+      togglbutton.deactivateAllTimerLinks();
+      return;
+    }
+
+    const current = Array.from(document.querySelectorAll('.toggl-button:not(.toggl-button-edit-form-button)'))
+      .find(button => button.title.indexOf(entry.description) !== -1);
```

Comment:
Note: the `title` attribute is set to `description + (project if project)` at insertion time, this seems to be the cheapest way to check if it's the correct entry. Shouldn't be necessary to run the description/project functions again and do loads of look ups (that would get quite intense with many links in a render-heavy service). 

There is perhaps a chance of an edge case where wrong entry is highlighted because it still matches in `indexOf`, but it seemed uncommon enough to be an acceptable compromise. Open to changing it.